app/src/main/java/org/oppia/android/app/devoptions/featureflags/FeatureFlagItemViewModel.kt

```diff
@@ -77,6 +79,7 @@ class FeatureFlagItemViewModel(
       SyncStatus.SYNC_STATUS_UNSPECIFIED -> 0xFF00645C
       SyncStatus.NOT_SYNCED_FROM_SERVER -> 0xFFBE563C
       SyncStatus.SYNCED_FROM_SERVER -> 0xFF00645C
+      SyncStatus.LOCAL_OVERRIDE -> 0xFFEFCF24
```

Comment:
Closer to this shade `0xFFC2B71B` makes it visible for the white text. (For light mode -- follow up if you have concerns with dark mode for the same)
```suggestion
      SyncStatus.LOCAL_OVERRIDE -> 0xFFC2B71B
```

Reply:
updated